src/Core/Core.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.0;netstandard2.0</TargetFrameworks>
```

Comment:
This is not required. You do not need to add a second platform if there is no specific api for it.

Reply:
Yep, removing this seemed to be OK; build and tests. Since we don't need to toggle with compiler flags for this framework or pull in different references.